mycore-pi/src/main/java/org/mycore/pi/urn/rest/MCRDNBURNRestClient.java

```diff
@@ -161,6 +164,36 @@ public Optional<Date> register(MCRPIRegistrationInfo urn) {
         return Optional.empty();
     }
 
+    public Optional<JsonObject> getRegistrationInfo(MCRPIRegistrationInfo urn) {
+        String identifier = urn.getIdentifier();
+        return getRegistrationInfo(identifier);
+    }
+
+    public static Optional<JsonObject> getRegistrationInfo(String identifier) {
+        MCRDNBURNRestClient mcrurnClient = new MCRDNBURNRestClient(
+                u -> MCRURNJsonBundle.instance(u, MCRDerivateURNUtils.getURL(u)));
+        String url = mcrurnClient.getBaseServiceURL() + "/urn/" + identifier;
+        CloseableHttpResponse response = MCRHttpsClient.get(url, Optional.empty());
+
+        int statusCode = response.getStatusLine().getStatusCode();
+        switch (statusCode) {
+            case HttpStatus.SC_OK:
+                HttpEntity entity = response.getEntity();
+                try {
+                    Reader reader = new InputStreamReader(entity.getContent());
+                    JsonElement jsonElement = JsonParser.parseReader(reader);
+                    return Optional.of(jsonElement.getAsJsonObject());
+                } catch (Exception e) {
+                    LOGGER.error("Could not read Response from " + url);
+                }
+                break;
+            default:
+                mcrurnClient.logFailure("", response, statusCode, identifier, url);
```

Comment:
The method logFailure() does not rely on any field of the class so you could make it static or add a static variant which is used by the non static version.